packages/nestjs/src/index.ts

```
@@ -0,0 +1,318 @@
+    };
+
+@Injectable()
+export class HiveGatewayDriver<
+  TContext extends Record<string, any>,
+> extends AbstractGraphQLDriver<HiveGatewayDriverConfig<TContext>> {
+  private _gatewayRuntime: GatewayRuntime<TContext> | undefined;
+  private _subscriptionService?: GqlSubscriptionService;
+
+  constructor() {
+    super();
+  }
```

Biome (1.9.4)

[error] This constructor is unnecessary.

Unsafe fix: Remove the unnecessary constructor.


(lint/complexity/noUselessConstructor)